views/pages/home.pug

```diff
@@ -23,10 +23,11 @@ block content
                         | Send Gifts to
                         |
                         span.typing( data-wait='2000' data-words='["Foster Children", "Homeless Youth", "Kids in Emergency Shelters"]' )
-                    p.py-1.mb-0 Every child should feel valued and loved
-                    p.py-1.mb-5 Donate holiday gifts and school supplies to the kids in need
-                    a.button-outline.m-3( href='/howto' ) How It Works
-                    a.button-accent.m-3( href='/wishcards' ) See Wish Cards
+                    p.py-1.mb-0.mobile-hide Every child should feel valued and loved
+                    p.py-1.mb-4.mobile-hide Donate holiday gifts and school supplies to the kids in need
+                    .d-flex.justify-content-center.align-items-center.actions
+                        a.button-outline.m-2( href='/howto' ) How It Works
+                        a.button-accent.m-2( href='/wishcards' ) See Wish Cards
```

Comment:
```suggestion
                    p.py-1.mb-0.d-none.d-md-block Every child should feel valued and loved
                    p.py-1.mb-4.d-none.d-md-block Donate holiday gifts and school supplies to the kids in need
                    .d-md-flex.justify-content-center.align-items-center
                        a.btn.btn-outline-light.btn-lg.m-2.w-75( href='/howto' ) How It Works
                        a.btn.btn-secondary.btn-lg.m-2.w-75( href='/wishcards' ) See Wish Cards
```

Reply:
achieves the same and uses bootstrap stuff

![image](https://github.com/donategifts/donategifts/assets/44695100/7feae5b7-1bff-4816-88f4-647df947e769)

Reply:
![image](https://github.com/donategifts/donategifts/assets/44695100/f31eeada-cdd6-446f-8609-e745dfa3a1c4)

Reply:
but it's not the exact look I'm going for though. the buttons are not exact match